testIndex.js

```diff
@@ -1,44 +1,19 @@
-jasmine.DEFAULT_TIMEOUT_INTERVAL = 10000;
+'use strict';
 
+// Engine tests
 var testsContext = require.context('./src', true, /__tests__\/.*\.test\.jsx?$/);
 testsContext.keys().forEach(testsContext);
 
 // This is necessary for the coverage report to show all source files even when they're not
-// included by tests. However, the source files will throw errors if they have require() statements
-// in them since they're expecting Turbine's injected require function which is why we must
-// wrap in a try-catch. When the source files are required by tests, the tests inject everything
-// necessary for them to work properly.
+// included by tests. https://github.com/webpack-contrib/istanbul-instrumenter-loader/issues/15
 var srcContext = require.context('./src', true, /^((?!__tests__).)*\.jsx?$/);
-srcContext.keys().forEach(function(src) {
-  try {
-    srcContext(src);
-  } catch(e) {
-    // Do nothing.
-  }
```

Comment:
The try-catch, as described in the comments, isn't needed any longer. It was for requiring core module stuff. But, removing the try catch exposed an issue where, for example, `index.js` would try to run and `_satellite` wouldn't be defined and would then throw errors. The changes to turbine source were to resolve these issues.